src/com_tjvendors/site/models/vendor.php

```diff
@@ -462,6 +464,10 @@ public function save($data)
 				Factory::getDbo()->insertObject('#__vendor_client_xref', $client_entry);
 				$tjvendorsTriggerVendor->onAfterVendorSave($data, true);
 
+				// Plugin trigger
+				PluginHelper::importPlugin('tjvendors');
+				Factory::getApplication()->triggerEvent('tjVendorOnAfterVendorSave', array($data));
```

Comment:
dont triggers needs to be starting as `on`

Reply:
done